### DISCUSSION TITLE
想问下Doris表有哪些属性在那里看啊？

### DISCUSSION THREAD

**Discussion by Loveca** [apache/doris#11431] [A - General / Q&A]
想问下Doris表有哪些属性在那里看啊？比如表的name,primaryKey这些

**Comment by dataalive** [comment]
show database
show create table ?

**Comment by deemogsw** [comment]
Connect to doris by navicate.You can get more detail just like use mysql

**Comment by hf200012** [comment]
https://doris.apache.org/zh-CN/docs/sql-manual/sql-reference/Data-Definition-Statements/Create/CREATE-TABLE
